src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
+  padding: 0;
+
+  --link-color: #00acdc;
+  --bg-color: #fff;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  display: flex;
+  background-color: var(--bg-color);
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__list :first-child {
+  margin-left: 0;
+}
+
+.nav__list :last-child {
+  margin-right: 0;
+}
+
+.nav__item {
+  margin: 0 10px;
```

Comment:
Be consistent with your *vertical* margins (Add only top or only bottom margin, but not both). Here you have horizontal margins, which is fine, but the comment is to remind you to be cautious with vertical margins and avoid collapsing issues.